src/sql/src/rbac.rs

```diff
@@ -611,7 +651,7 @@ fn generate_rbac_requirements(
                 AclMode::USAGE,
                 role_id,
             )],
-            item_usage: false,
+            item_usage: BTreeSet::new(),
```

Comment:
If we are able to switch to a reference for this field, using named (even though empty) vars would be good.